test/metabase/driver/common/parameters/values_test.clj

```diff
@@ -135,57 +168,71 @@
+                 :display-name "Card template tag test"
+                 :type         :card
+                 :card-id      (:id card)}
                 []))))))
 
   (testing "Card query template tag generates native query for MBQL query"
     (qp.test-util/with-everything-store
       (driver/with-driver :h2
         (let [mbql-query   (data/mbql-query venues
                              {:database (data/id)
-                              :filter [:< [:field-id $price] 3]})
+                              :filter   [:< [:field-id $price] 3]})
               expected-sql (str "SELECT "
-                                  "\"PUBLIC\".\"VENUES\".\"ID\" AS \"ID\", "
-                                  "\"PUBLIC\".\"VENUES\".\"NAME\" AS \"NAME\", "
-                                  "\"PUBLIC\".\"VENUES\".\"CATEGORY_ID\" AS \"CATEGORY_ID\", "
-                                  "\"PUBLIC\".\"VENUES\".\"LATITUDE\" AS \"LATITUDE\", "
-                                  "\"PUBLIC\".\"VENUES\".\"LONGITUDE\" AS \"LONGITUDE\", "
-                                  "\"PUBLIC\".\"VENUES\".\"PRICE\" AS \"PRICE\" "
+                                "\"PUBLIC\".\"VENUES\".\"ID\" AS \"ID\", "
+                                "\"PUBLIC\".\"VENUES\".\"NAME\" AS \"NAME\", "
+                                "\"PUBLIC\".\"VENUES\".\"CATEGORY_ID\" AS \"CATEGORY_ID\", "
+                                "\"PUBLIC\".\"VENUES\".\"LATITUDE\" AS \"LATITUDE\", "
+                                "\"PUBLIC\".\"VENUES\".\"LONGITUDE\" AS \"LONGITUDE\", "
+                                "\"PUBLIC\".\"VENUES\".\"PRICE\" AS \"PRICE\" "
                                 "FROM \"PUBLIC\".\"VENUES\" "
                                 "WHERE \"PUBLIC\".\"VENUES\".\"PRICE\" < 3 "
                                 "LIMIT 1048576")]
           (tt/with-temp Card [card {:dataset_query mbql-query}]
             (is (= (i/->ReferencedCardQuery (:id card) expected-sql)
                    (#'values/value-for-tag
-                    {:name "card-template-tag-test", :display-name "Card template tag test",
-                     :type :card, :card-id (:id card)}
+                    {:name         "card-template-tag-test"
+                     :display-name "Card template tag test"
+                     :type         :card
+                     :card-id      (:id card)}
                     []))))))))
 
   (testing "Card query template tag wraps error in tag details"
     (tt/with-temp Card [param-card {:dataset_query
                                     (data/native-query
-                                     {:query "SELECT {{x}}"
-                                      :template-tags
-                                      {"x"
-                                       {:id "x-tag", :name "x", :display-name "Number x",
-                                        :type :number, :required false}}})}]
+                                      {:query "SELECT {{x}}"
```

Comment:
Why the extra space indent? Doesn't [this rule](https://guide.clojure.style/#one-space-indent) apply?

Reply:
IMO `data/native-query` is a macro with a body so the 2-space-indent rule applies to it. I guess it could really go either way but I put `:style/indent 2` metadata on it a while back so that's just what CIDER does for me automatically